extensions/dicom-microscopy/src/index.tsx

```diff
@@ -90,6 +90,45 @@ export default {
     ];
   },
 
+  getToolbarModule({ servicesManager }) {
+    return [
+      {
+        name: 'evaluate.microscopyTool',
```

Comment:
This use here demonstrates that the toolbars are now extension independent and belong in the UI group instead of the default extension.